presto-main/src/main/java/com/facebook/presto/sql/gen/LambdaAndTryExpressionExtractor.java

```diff
@@ -97,6 +98,12 @@ public Void visitVariableReference(VariableReferenceExpression reference, Contex
             return null;
         }
 
+        @Override
+        public Void visitDeferredSymbolReference(DeferredSymbolReferenceExpression reference, Context context)
```

Comment:
Separate commit introducing `DeferredSymbolReferenceExpression`, and explaining in commit message what it is would be nice.